docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/snippets/basemanifest.bicep

```diff
@@ -0,0 +1,65 @@
+import radius as radius
+
+@description('Specifies the location for resources.')
+param location string = 'global'
+
+@description('Specifies the image of the container resource.')
+param magpieimage string = 'ghcr.io/radius-project/dev/magpiego:pr-498da3203f'
+
+@description('Specifies the port of the container resource.')
+param port int = 3000
+
+@description('Specifies the environment for resources.')
+param environment string
+
+@description('Specifies the application for resources.')
+param application string
+
+@description('Specifies the namespace for resources.')
+param namespace string = 'my-microservice'
+
+//APPLICATION
+resource app 'Applications.Core/applications@2023-10-01-preview' = {
+  name: application
+  location: location
+  properties: {
+    environment: environment
+    extensions: [
+      {
+          kind: 'kubernetesNamespace'
+          namespace: namespace
+      }
+    ]
+  }
+}
```

Comment:
No need to define the application in this file. Let's omit this and not use the namespace and then specify the app name as part of `rad deploy -a`